pkg/cmd/server/start/start_allinone.go

```diff
@@ -61,12 +57,8 @@ You may also pass --kubeconfig=<path> to connect to an external Kubernetes clust
 
 // NewCommandStartAllInOne provides a CLI handler for 'start' command
 func NewCommandStartAllInOne(fullName string, out io.Writer) (*cobra.Command, *AllInOneOptions) {
```

Comment:
Renamed `fullName`.  It isn't a full name in the same sense as our other commands and now that it has a semantic meaning (used below in `DefaultsFromName`).